src/webgpu/shader/execution/expression/unary/unary.ts

```diff
@@ -0,0 +1,6 @@
+import { ExpressionBuilder } from '../expression.js';
+
+/* @returns an ExpressionBuilder that evaluates the prefix unary operation */
+export function prefix(op: string): ExpressionBuilder {
```

Comment:
In other expressions we've called this helper the name of the directory, i.e. `builtin`, `binary`, etc.